src/parser/main.js

```
@@ -50,4 +50,132 @@ function replace(str, replacements) {
+            case 'heredoc':
+                return `${fd}<<< ${target}`;
+            default:
+                return '';
+        }
+    }
+
+    // Recursively render a command plus any piped successors.
+    function renderCommand(cmd) {
+        let parts = [];
+
+        // Component (executable name)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'parts' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)